nest_elephant_tvb/translation/FatEndPoint.py

```diff
@@ -0,0 +1,196 @@
+from mpi4py import MPI
+import pathlib
+
+def make_connections(path_to_files_receive, path_to_files_send, logger_master):
+    '''
+    Fat end point, here: connections to nest and tvb.
+    '''
+    use_mpi = True
+    # init MPI
+    if use_mpi:
+        comm = MPI.COMM_WORLD  # INTRA communicator
+        info = MPI.INFO_NULL
+        root=0
+    else:
+        comm = None
+    ### nest connection
+    comm_receiver, port_receive = nest_connection(comm, root, info, logger_master, path_to_files_receive)
+    ### tvb connecction
+    comm_sender, port_send = tvb_connection(comm, root, info, logger_master, path_to_files_send)
+    logger_master.info('Connections made, starting translation...')
+    
+    return comm_receiver, port_receive, comm_sender, port_send
+
+
+def nest_connection(comm, root, info, logger_master, path_to_files_receive):
+    '''
+    MPI inter communicator to NEST
+    
+    '''
+    logger_master.info('Translate Receive: before open_port')
+    if comm.Get_rank() == 0:
+        ### Connection to simulation (incoming data)
+        port_receive = MPI.Open_port(info)
+        logger_master.info('Translate Receive: after open_port: '+port_receive)
+        # Write file configuration of the port
+        fport = open(path_to_files_receive, "w+")
+        fport.write(port_receive)
+        fport.close()
+        pathlib.Path(path_to_files_receive+'.unlock').touch()
+        logger_master.info('Translate Receive: path_file: ' + path_to_files_receive)
+    else:
+        port_receive = None
+    
+    # broadcast port info, accept connection on all ranks!
+    # necessary to avoid problems with information about the mpi rank in open port file.
+    port_receive = comm.bcast(port_receive,root)
+    logger_master.info('Translate Receive: Rank ' + str(comm.Get_rank()) + 'accepting connection on: ' + port_receive)
+    comm_receiver = comm.Accept(port_receive, info, root) 
+    logger_master.info('Translate Receive: Simulation client connected to' + str(comm_receiver.Get_rank()))
+    
+    return comm_receiver, port_receive
+
+
+def tvb_connection(comm, root, info, logger_master, path_to_files_send):
+    '''
+    MPI inter communicator to TVB
+    
+    '''
+    logger_master.info('Translate SEND: before open_port')
+    if comm.Get_rank() == 0:
+        ### Connection to simulation (incoming data)
+        port_send = MPI.Open_port(info)
+        logger_master.info('Translate SEND: after open_port : '+port_send)
+        # Write file configuration of the port
+        fport = open(path_to_files_send, "w+")
+        fport.write(port_send)
+        fport.close()
+        pathlib.Path(path_to_files_send+'.unlock').touch()
+        logger_master.info('Translate SEND: path_file: ' + path_to_files_send) # ??? ;sys.stdout.flush()
```

Comment:
Remove spurious source code comment

Reply:
```suggestion
        logger_master.info('Translate SEND: path_file: ' + path_to_files_send)
```
I removed the comment. It was a leftover from the old code (and did not seem to serve any purpose).